main.nf

```diff
@@ -1360,8 +1427,81 @@ process circularmapper{
     
 }
 
+process bowtie2 {
+    label 'mc_medium'
+    tag "${libraryid}"
+    publishDir "${params.outdir}/mapping/bt2", mode: 'copy'
+
+    input:
+    tuple samplename, libraryid, lane, seqtype, organism, strandedness, udg, file(r1), file(r2) from ch_lanemerge_for_bt2
+    file index from bt2_index.collect()
+
+    output:
+    tuple samplename, libraryid, lane, seqtype, organism, strandedness, udg, file("*.mapped.bam"), file("*.{bai,csi}") into ch_output_from_bt2
+    tuple samplename, libraryid, lane, seqtype, organism, strandedness, udg, file("*_bt2.log") into ch_bt2_for_multiqc
+
+    when: 
+    params.mapper == 'bowtie2'
+
+    script:
+    size = "${params.large_ref}" ? '-c' : ''
+    fasta = "${index}/${bt2_base}"
+    trim5 = "${params.bt2_trim5}" != 0 ? "--trim5 ${params.bt2_trim5}" : ""
+    trim3 = "${params.bt2_trim3}" != 0 ? "--trim3 ${params.bt2_trim3}" : ""
+    bt2n = "${params.bt2n}" != 0 ? "-N ${params.bt2n}" : ""
+    bt2l = "${params.bt2l}" != 0 ? "-L ${params.bt2l}" : ""
```

Comment:
Update default

Reply:
I've addressed this in the docs and doesn't require update because it overrides the default from `--bt2_sensitivity`.